Privacy-policy-snippets.md

```diff
@@ -0,0 +1,36 @@
+# Privacy Policy Snippets
+
+Cut and paste-able snippets that describe how core collects, manages and shares data provided by site users and visitors.
+
+## Definitions
+
+* Logged In vs Logged Out Users: Logged in users are users who have registered (created) an account on the site and have provided their username and password to log in to the site. Some sites require users to be registered and logged in to do things like comment on a post. Logged out users are users who have not logged in, or who do not have an account on the site
+at all (e.g. visitors)
```

Comment:
Looks like maybe a rogue line break here and period missing on end of sentence.

Reply:
Fixed in https://github.com/gdpr-compliance/info/pull/8/commits/3481f3b64f5138e52a3fbfc2c06c6d6c9c14859b